pool-interface/src/lib.rs

```diff
@@ -89,4 +93,6 @@ pub trait LendingPoolTrait {
     fn set_liq_index(env: Env, asset: Address, value: i128) -> Result<(), Error>;
 
     fn borrow(env: Env, who: Address, asset: Address, amount: i128) -> Result<(), Error>;
+
+    fn set_pause(env: Env, value: bool) -> Result<(), Error>;
```

Comment:
Usually, when implementing pausable pattern, devs also expose "paused" method for the end-users to check if the contract is paused. What do you think about it? Should we also add it?

Reply:
Why not, will add it